src/features/home/atoms/OfferTile.tsx

```diff
@@ -20,14 +22,31 @@ interface OfferTileProps {
   layout?: Layout
 }
 
+export function useCrashQuery() {
+  return useQuery(
+    'crash',
+    async function () {
+      await new Promise((res) => setTimeout(res, 1000))
+      return await getExternal('crash')
+    },
+    {
+      enabled: false,
+      retry: false,
+    }
+  )
+}
+
```

Comment:
Si ce commit est temporaire/ not to be merged, comment les PO/UX pourront tester la feature ?
Peut être un crash au niveau de la page Cheatcodes ?

Reply:
en se mettant en mode avion ça devrait marcher. Mais il faut que je mette en place un loader en attendant (skeleton)